examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Waiting for Kafka on port 9092"
+wait_for 120 sh -c "docker-compose logs -f kafka | grep -q 'started (kafka.server.KafkaServer)'"
+run_log "Kafka launched"
+
+# Topic
+TOPIC="envoy.kafka.broker"
+run_log "Creating topic"
+docker-compose exec -T kafka kafka-topics --bootstrap-server localhost:19092 --create --topic $TOPIC quickstart-events
+
+run_log "Checking topic"
+topic_created=$(docker-compose exec -T kafka kafka-topics --bootstrap-server localhost:19092 --list)
+if [[ "$topic_created" == "$TOPIC" ]]; then
```

Comment:
how could this every be in more than one state?

Reply:
Just in case, we cannot create the `TOPIC` successfully, and `verify.sh` will exit with `exit 1`.

Reply:
there should be no just in case - it needs to work

Reply:
Because from my testing result, rarely we get the timeout error when creating a topic. So I add this logic to make sure the TOPIC was created before we continue to do something else.

Reply:
either way it needs to work reliably - otherwise it will create flakes in CI